src/Proto.Actor/Context/RootContext.cs

```diff
@@ -70,6 +70,25 @@ public PID SpawnNamed(Props props, string name)
             throw;
         }
     }
+    
+    public PID SpawnNamedSystem(Props props, string name)
```

Comment:
this should be moved somewhere else to not clutter RootContext.
Maybe an extension method

Reply:
`SpawnNamedSystemActor` ?